src/main/java/io/kontur/disasterninja/dto/UserMetricDto.java

```diff
@@ -0,0 +1,19 @@
+package io.kontur.disasterninja.dto;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class UserMetricDto {
+
+    private final String name;
+    private final Double value;
+    private final UserMetricDtoType type;
+    private final UUID appId;
+    private final UUID userId;
+
+    public enum UserMetricDtoType {
+        summary
```

Comment:
This is out of java convention, personally I would have labeled SUMMARY("summary") here